core/src/main/java/google/registry/schema/domain/RegistryLock.java

```diff
@@ -57,11 +57,13 @@
      * <p>Note: because of this index, physical columns must be declared in the {@link Column}
      * annotations for {@link RegistryLock#revisionId} and {@link RegistryLock#repoId} fields.
      */
-    indexes =
-        @Index(
-            name = "idx_registry_lock_repo_id_revision_id",
-            columnList = "repo_id, revision_id",
-            unique = true))
+    indexes = {
+      @Index(
+          name = "idx_registry_lock_repo_id_revision_id",
+          columnList = "repo_id, revision_id",
+          unique = true),
+      @Index(name = "idx_registry_lock_verification_code", columnList = "verificationCode")
```

Comment:
Of note: it seems like Hibernate actually requires "verificationCode" here, not "verification_code". It failed to create this index in the schema tests otherwise.

Reply:
btw it's because until it gets finally written out, Hibernate apparently treats the column as "verificationCode" whereas the Column annotations are applied pretty much immediately (this is why repo_id and revision_id work in the previous index annotation)